src/GenAPI/Microsoft.DotNet.GenAPI.Tool/Microsoft.DotNet.GenAPI.Tool.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">net7.0</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFramework>net6.0</TargetFramework>
    <TargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net7.0</TargetFramework>
```

Reply:
As GenAPI is not part of the SDK, yes. But ApiSymbolExtensions and anything ApiCompat related needs to stay on net6.0 and net472.

Reply:
Just in case: I noticed, that ApiContat.Tool targets net6.0 only